lib/pages/getStaticProps/setup.js

```diff
@@ -21,6 +23,16 @@ const setup = () => {
     // Copy page's JSON data
     const jsonPath = getFilePathForRoute(route, "json");
     setupStaticFileForPage(jsonPath, dataRoute);
+
+    // Set up the Netlify function (this is ONLY for preview mode)
+    if (srcRoute) {
+      // is this taken care of in the other getStaticProps steps? unclear
```

Comment:
Nope! `srcRoute` just indicates whether a page uses dynamic routing and `getStaticPaths`. 

We take care of **some** pages with `srcRoute`, namely those that have `fallback: true` or have a `revalidation` interval. A page without fallback or revalidation would need to be set up!

I'm not sure about the cleanest way to go about this...

Reply:
oof ok that adds complexity (or just validates the complexity i was already facing 😆 )